src/CONST.ts

```diff
@@ -2873,6 +2873,7 @@ const CONST = {
              */
             NVP_FINANCIALFORCE_EXPORT_VENDOR_POLICY_ID: 'financialforce_export_vendor_policy_id',
         },
+        CARD_NAME_INPUT_LIMIT: 100,
```

Comment:
@narefyev91 Should we use CARD_TITLE_INPUT_LIMIT for consistency

Reply:
yeah we can - i thought that generally we adding card name and not title.... @DylanDylann That's why make a const based on what exact action we are executing

Reply:
@narefyev91 I think the card name and card title have the same meaning

Reply:
@narefyev91 About the limit character, we can ask design team to confirm if we should use 100 or 255. And then apply it for both Company Card and Expensify Card

Reply:
yeah in design doc - it was a requirement that card name should be <=100 characters